style.css

```diff
@@ -379,3 +381,130 @@ ul li {
   line-height: 24px;
   color: #000;
 }
+
+/* Contact */
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 48rem;
+  width: 375px;
+  border-top-left-radius: 60px;
+  background-image: url("media/header-shapes\ mobile@2x.svg");
+  background-size: cover;
+  background-color: #6070ff;
+}
```

Comment:
- [ ] You have done a great job, but to ensure that your project matches the Figma template accurately, I suggest using the same background image as specified in the design. Doing so will ensure consistency in the visual design of your project and improve its overall appearance.

**Your design**

![Screenshot from 2023-04-06 07-23-25](https://user-images.githubusercontent.com/105117832/230378013-d2f186aa-b5a4-4ad8-beb0-9dadbf95cab3.png)

**Figma**

![Screenshot from 2023-04-06 07-23-00](https://user-images.githubusercontent.com/105117832/230378087-a276f219-6284-41e8-9482-54e4f3748b4a.png)